docs/at_variables_reference.rst

```diff
@@ -0,0 +1,93 @@
+.. _AtVariables:
+
+@-Variables
+===========
+
+@-variables are placeholder variables that can be used in the ``experiments.yml`` file. They are enclosed
+by at signs, i.e. ``@...@`` and get resolved during runtime. Some use cases are:
+
+- allowing variations of experiments
+- accessing directories, which are created by simexpal during runtime
+- accessing values for keys specified in the ``experiments.yml``
+
+..
+    TODO: Add section Instances for generators and INSTANCE_FILENAME variable
+
+In the sections below we will list all @-variables and where they can be used.
```

Comment:
Maybe it makes sense to list all @ variables first and then list for each variable where it will be used.

Reply:
Isn't it better to list for all keys the variables that can be used (as it is right now)? And not the other way around? I think that makes it easier to find the options you have for a key.

I think it is a good idea to list all @-variables beforehand though.